terraform/lambdas.tf

```diff
@@ -1,9 +1,15 @@
 resource "aws_security_group" "keyconjurer-default" {
-  name        = "Key Conjurer ${terraform.workspace}"
+  name_prefix = "keyconjurer"
   description = "default security group to allow most expected protocols"
   vpc_id      = var.vpc_id
 
-  tags = var.tags
+  egress {
```

Comment:
I am wondering how it used to work. From the TF docs

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group

> By default, AWS creates an ALLOW ALL egress rule when creating a new Security Group inside of a VPC. When creating a new Security Group inside a VPC, Terraform will remove this default rule, and require you specifically re-create it if you desire that rule.

I guess it's enforced in newer TF versions. Which one do you use?

Reply:
That behavior has been in place since at least Terraform 0.12; KeyConjurer was originally configured to use 0.13 and now uses 1.3.7 I think.